view/widgets/acquisition_widgets/volume_widget.py

```diff
@@ -451,6 +453,22 @@ def toggle_z_show(self, current_row, current_column, previous_row, previous_colu
             z = self.scan_plan_widget.z_plan_widgets[show_row, show_col]
             self.scan_plan_widget.stacked_widget.setCurrentWidget(z)
 
+    def handle_snapshot(self, image):
+        """Correctly contrast image and grab fov coordinates
+        :param image: numpy array of image to display in model"""
+
+        # grab correct coordinates where image was taken
+        coords = self.fov_position
+
+        # contrast image
+        maximum = np.percentile(image, 90)
+        minimum = int(np.percentile(image, 5))
```

Comment:
The percentiles might need to be tuned per system/specimen type. Not sure if it makes sense to pull these out as static values in the YAML file. Or to have control widgets. YAML is probably sufficient, widget might be overkill. I would also change from percentile to absolute intensity. For example I know when live previewing on the exaspim that [30, 100] as min/max yield good visibility for a human operator.

I would also modify the display of the uint16 image as opposed to the image itself. I think this could be set at the GL level, see other comment.